_layouts/default.html

```diff
@@ -106,6 +106,8 @@ <h3>Exchanges</h3>
           <ul class="quick-links">
             <li><a href="https://bittrex.com/Market/Index?MarketName=BTC-AEON" rel="nofollow">Bittrex</a></li>
             <li><a href="https://hitbtc.com/exchange/AEON-to-BTC" rel="nofollow">HitBTC</a></li>
+            <li><a href="https://tradeogre.com/exchange/BTC-AEON" rel="nofollow">TradeOgre</a></li>
+            <li><a href="https://bitliber.com/markets/aeon-btc" rel="nofollow">Bitliber</a></li>
```

Comment:
Perhaps also mention the AEON-XMR pair that's available in Bitliber?

Reply:
I gave thought to this but I was not sure how to solve this pretty.
but yes you are right, it would be cool to add bitliber aeon/xmr pair too.

What do you think about the following preview?

![download](https://user-images.githubusercontent.com/39473497/43005785-29474b62-8c34-11e8-9c7c-64d1ee5c629b.png)

Reply:
It looks good to me. At least much better than not having it at all.